src/functionaltests/java/com/ericsson/ei/threadingAndWaitlistRepeat/ThreadingAndWaitlistRepeatSteps.java

```diff
@@ -107,7 +106,10 @@ public void correct_amount_of_threads_should_be_spawned() throws Throwable {
 
     @Then("^after the time to live has ended, the waitlist should be empty$")
     public void after_the_time_to_live_has_ended_the_waitlist_should_be_empty() throws Throwable {
-        TimeUnit.SECONDS.sleep(waitlistTtl + 10);
+        long stopTime = System.currentTimeMillis() + 120000;        
```

Comment:
change to 

System.currentTimeMillis() + TimeUnit.SECONDS.toMillis(waitlistTtl + 60);